matlab/schemas/+anatomy/AreaMask.m

```diff
@@ -0,0 +1,65 @@
+%{
+# Area mask for each scan
+-> experiment.Scan
+-> anatomy.Area
+field                    : tinyint               # slice id
+---
+mask                     : mediumblob            # mask of area
+%}
+
+classdef AreaMask < dj.Imported
+    methods(Access=protected)
+        function makeTuples(obj,key) %create clips
+            insert( obj, key );
+        end
+    end
+    
+    methods
+        
+        function plot(obj)
+            
+            [masks,areas] = fetchn(obj,'mask','area');
+            vessels = fetchn(map.OptImageBar & obj,'vessels');
+            
+            area_map = zeros(size(masks{1}));
+            for imasks = 1:length(masks)
+                area_map(masks{imasks}) = imasks;
+            end
+            
+            im(:,:,1) = normalize(area_map);
```

Comment:
`cat(3, ...)` may be more explicit here.

Reply:
Fixed